cluster/addons/e2e-rbac-bindings/admin-binding.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+kind: ClusterRoleBinding
+metadata:
+  name: admin-cluster-admin
+  labels:
+    kubernetes.io/cluster-service: "true"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: view
+subjects:
+- apiVersion: rbac/v1alpha1
+  kind: User
+  name: admin
```

Comment:
a comment in these files about what they are needed for, and whether they are an acceptable end state or have a TODO to be removed/tightened might be helpful